internal/fileio/fileio.go

```diff
@@ -36,16 +36,16 @@ import (
 func ParseHalConfig(halPath string) (*config.Hal, error) {
 	data, err := ioutil.ReadFile(halPath)
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("unable to read %q error: %v", halPath, err)
```

Comment:
```suggestion
		return nil, fmt.Errorf("unable to read %q: %v", halPath, err)
```
or:
```suggestion
		return nil, fmt.Errorf("unable to read %q. Error: %v", halPath, err)
```
No strong opinion other than these be relatively consistent and the actual error message is cleanly separated from the custom message.

Reply:
@achew22 I ended up just committing one of these suggestions so there's some separation between the error message, so that we can get this merged. Thanks again for the contribution!